vdf/src/proof_of_time.rs

```diff
@@ -12,14 +12,13 @@
 // See the License for the specific language governing permissions and
 // and limitations under the License.
 use super::{classgroup::ClassGroup, proof_pietrzak::verify_proof};
-use gmp_classgroup::ffi::import_obj;
 use sha2::{digest::FixedOutput, Digest, Sha256};
 use std::collections::HashMap;
 use std::usize;
 
 fn generate_r_value<T>(x: &T, y: &T, sqrt_mu: &T, int_size_bits: usize) -> Result<T::BigNum, ()>
 where
-    T: ClassGroup<BigNum = super::gmp_classgroup::Mpz>,
+    T: ClassGroup,
```

Comment:
Should the other two bounds below be part of the `ClassGroup` trait?

Reply:
I see, they're already there in the `where` clause… I guess that doesn't work then. :confused:

Reply:
Correct, sadly.